packages/calcite-components/src/components/time-picker/time-picker.scss

```diff
@@ -1,17 +1,32 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-time-picker-background-color: The background color of the time picker.
+ * @prop --calcite-time-picker-corner-radius: The border radius of the time picker.
+ * @prop --calcite-time-picker-button-background-color-hover: Specifies the button's background color when hovered or focused.
+ * @prop --calcite-time-picker-button-background-color-press: Specifies the button's background color when active.
+ * @prop --calcite-time-picker-color: Specifies the component's background color.
```

Comment:
The description of this token sounds duplicative to `--calcite-time-picker-background-color`.

Reply:
Looks like it should say something like `specifies the component's text color`.